tests/Fixer/Operator/BinaryOperatorSpacesFixerTest.php

```diff
@@ -2051,6 +2051,74 @@ function foo() {}
     }}, 2 => 3];
 ',
             ],
+            [
+                '<?php
+                    $result = $this->test1(fn () => $this->test2($a, $b));
+                    foreach ($result as $k => $v)
+                    {
+                        $bv = [
+                            $b  => 2,
+                            $cc => 3,
+                        ];
+                    }
+                    fn ($x) => $x + 1 ?>',
+                '<?php
+                    $result = $this->test1(fn () => $this->test2($a, $b));
+                    foreach ($result as $k => $v)
+                    {
+                        $bv = [
+                            $b => 2,
+                            $cc => 3,
+                        ];
+                    }
+                    fn ($x) => $x + 1 ?>',
+            ],
+            [
+                '<?php
+                $zzzzzz
+                    ->rtrim()
+                    ->filter(fn (string $foo) => $foo."1")
+                    ->reduce(fn (string $bar123) => $foo - 1)
+                ;
+
+                $array = [
+                    \'foo\'     => 123,
+                    \'longkey\' => \'test\',
+                    \'baz\'     => fn () => \'value\',
+                ];
+
+                function share(Request $request): array
+                    {
+                        return array_merge(parend::share($request), [
+                            \'flashNotifications\' => fn () => Session::get(\'flash_notification\'),
+                            \'currentUser\'        => fn () => $thiz->getCurrentUser($request->user()),
+                            \'menuItems\'          => fn () => $thiz->getMenuItems($request->user()),
+                        ]);
+                    }
+                ',
+                '<?php
+                $zzzzzz
+                    ->rtrim()
+                    ->filter(fn (string $foo) => $foo."1")
+                    ->reduce(fn (string $bar123) => $foo - 1)
+                ;
+
+                $array = [
+                    \'foo\' => 123,
+                    \'longkey\' => \'test\',
+                    \'baz\' => fn () => \'value\',
+                ];
+
+                function share(Request $request): array
```

Comment:
There is no example similar to
```
  \DB::transaction(function () use ($entity) {
            $goalFile = new GoalFile();
            $goalFile->goal_id = $entity->goalId()->value();
            $goalFile->author_id = $entity->authorId()->value();
            $goalFile->original_name = $entity->name();
            $goalFile->description = $entity->description();

            $file = new File();
            $file->storage = $entity->attributes()->storage();
            $file->directory = $entity->attributes()->directory();
            $file->file_name = $entity->attributes()->fileName();
            $file->mime_type = $entity->attributes()->mimeType();
            $file->size = $entity->attributes()->size();
            $file->save();

            $goalFile->file()->associate($file);
            $goalFile->save();
        });
```
in the tests

If it doesn't align `=` inside function, this shouldn't be considered as solving https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5667

Reply:
I think you have a different idea of what the spec should be than others, including people who added tests before you. That is fine, but I'm not going to try to make something I don't agree with.